packages/fast-components-react-msft/src/context-menu-item/examples.data.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import { ContextMenuItem, ContextMenuItemProps } from "./index";
+import { ComponentFactoryExample } from "@microsoft/fast-development-site-react";
+import schema from "@microsoft/fast-components-react-base/dist/context-menu-item/context-menu-item.schema.json";
+import Documentation from "./.tmp/documentation";
+
+export default {
+    name: "Context menu item",
+    component: ContextMenuItem,
+    schema: schema as any,
+    documentation: <Documentation />,
+    detailData: {
+        children: "wee",
```

Comment:
nit: should we maybe use "Hello world", "Context menu item" or "Foo" etc?

Reply:
lol you don't like "wee"?